Composer/packages/client/src/components/QnA/ReplaceQnAFromModal.tsx

```diff
@@ -395,20 +396,19 @@ export const ReplaceQnAFromModal: React.FC<ReplaceQnAModalProps> = (props) => {
                 <div style={descriptionStyle}>
                   {formatMessage('Select this option when you want to import existing KB from QnA maker portal. ')}
                 </div>
-                {!shouldProvideTokens &&
-                  (signedInAccount ? (
-                    <div style={accountInfo}>
-                      <span>{`Signed in as ${signedInAccount}. Click `}</span>
-                      <span style={signInButton} onClick={performNextAction}>
-                        {'next '}
-                      </span>
-                      <span>{'to select KBs'}</span>
-                    </div>
-                  ) : (
-                    <div style={signInButton} onClick={performNextAction}>
-                      {formatMessage('Sign in to Azure to continue')}
-                    </div>
-                  ))}
+                {signedInAccount ? (
```

Comment:
maybe wrap this piece of UI into it's own component and reuse it (+ same comments as above)